device/dhcp/clients/dhcpclient.yaml

```diff
@@ -0,0 +1,20 @@
+components:
+  schemas:
+    Dhcp.client:
+      description: >-
+        Configuration for emulated DHCPv4 or DHCPv6 Client on a single Interface.
+        https://www.rfc-editor.org/rfc/rfc2131.html
+      type: object
+      x-status:
+        status: under_review
+      properties:
```

Comment:
IxNetwork sample has one use case called `Multiple DHCP Client over same MAC`. SM expert may comment on it how user frequently use this option. 
Otherwise we have to support `Array` similar as ` ipv4_addresses`

This is IxNetwork snapshot:
![image](https://github.com/open-traffic-generator/models/assets/35574510/b7e48e0c-ed75-4f2b-8832-3e2efbff8476)